src/Model/Fulfilment/AbstractOrder.php

```diff
@@ -0,0 +1,289 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\src\Model\Fulfilment;
+
+use DateTime;
+use MyParcelNL\Sdk\src\Adapter\DeliveryOptions\AbstractDeliveryOptionsAdapter;
+use MyParcelNL\Sdk\src\Model\BaseModel;
+use MyParcelNL\Sdk\src\Model\Recipient;
+use MyParcelNL\Sdk\src\Support\Collection;
+
+class AbstractOrder extends BaseModel
+{
+    public const DATE_FORMAT_FULL = 'Y-m-d';
+
+    /**
+     * @var \MyParcelNL\Sdk\src\Adapter\DeliveryOptions\DeliveryOptionsFromOrderAdapter
+     */
+    protected $delivery_options;
```

Comment:
Uiteraard vraag ik mij af waarom deze allemaal snake_case zijn, wat is de regel?

Reply:
Dit doen we ook al in andere classes en in de API, is wmb dan gewoon een standaard die we dan eigenlijk nog even moeten vastleggen